pkg/reconciler/broker/broker.go

```diff
@@ -145,45 +134,12 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, b *eventingv1.Broker) pk
 		return err
 	}
 
-	if isUsingOperator(b) {
-		args.RabbitMQCluster = b.Spec.Config.Name
-		return r.reconcileUsingCRD(ctx, b, args)
-	} else {
-		return r.reconcileUsingLibraries(ctx, b, args)
+	if !isUsingOperator(b) {
+		// TODO: We need to flag this as an error. Or, maybe it comes from above (getExchangeArgs)
```

Comment:
I created an issue last week here:
https://github.com/knative-sandbox/eventing-rabbitmq/issues/304

If we do that, then we could just validate differently for both of these, so we only accept what we can support. I think it's better anyways so the resource fails fast instead of during runtime, wdyt?